packages/rum-core/src/browser/cookieObservable.spec.ts

```diff
@@ -38,22 +37,8 @@ describe('cookieObservable', () => {
     clock.tick(WATCH_COOKIE_INTERVAL_DELAY)
   })
 
-  it('should notify observers on cookie change when cookieStore is not supported', () => {
-    Object.defineProperty(window, 'cookieStore', { get: () => undefined, configurable: true })
-    const observable = createCookieObservable(mockRumConfiguration(), COOKIE_NAME)
-
-    let cookieChange: string | undefined
-    subscription = observable.subscribe((change) => (cookieChange = change))
-
-    setCookie(COOKIE_NAME, 'foo', COOKIE_DURATION)
-    clock.tick(WATCH_COOKIE_INTERVAL_DELAY)
-
-    expect(cookieChange).toEqual('foo')
-  })
-
-  it('should not notify observers on cookie change when the cookie value as not changed when cookieStore is not supported', () => {
-    Object.defineProperty(window, 'cookieStore', { get: () => undefined, configurable: true })
-    const observable = createCookieObservable(mockRumConfiguration(), COOKIE_NAME)
+  it('should not notify observers on cookie change when the cookie value as not changed', () => {
```

Comment:
Nit: looks like there was a typo in the original name of the test:
```suggestion
  it('should not notify observers on cookie change when the cookie value has not changed', () => {
```